contents/blog/measuring-feature-success.md

```diff
@@ -0,0 +1,132 @@
+
+![Message](../images/blog/measuring-feature-success/message.png)
+
+A feature is successful at this stage if users use it and the qualitative feedback we receive from them is positive. A common goal at this stage is "5 happy teams using it."
+
+### 3. Launch and monitor usage
+
+When we have positive feedback and usage in beta, and the feature reaches the point we are ready to release. The flag rolls out further, eventually to 100% of users.
+
+Yakko continued to update the simple UI for sampling (it changed four times) to ultimately land on something he was happy with. He instrumented tracking for the feature’s usage and rolled it out to 25% of all organizations in PostHog.
+
+![Sampling UI](../images/blog/measuring-feature-success/sampling.png)
+
+At this stage, success is significant usage by the target user. What this portion is depends on the usage of similar features. At PostHog, this looks like:
+
+- 10-12% of our daily active users use individual products (like product analytics or session recordings)
+- 15-40% of daily active product users use a large features (like flag payloads or funnels) within those products.
+- 4-12% of daily active product users use a small features (like sampling, replay network tab, or dashboard templates) within those products.
+
+The feature should also be issue free and work as intended. Bug reports or confusion on how to use the feature are bad signs.
+
+To evaluate success after launch, we watch session replays and track metrics that match specific criteria we care about. For example, we might look at the session recordings for high ICP, paid users in the 99th percentile of feature usage to see how power users are utilizing the product. We also might create a dashboard to show usage and the feature’s impact (such as average insight load time).
+
+In Yakko’s case, he: 
+
+- created a key metrics dashboard that he would dig into a few times a day.
+- watched [session replays](/tutorials/explore-insights-session-recordings) of the feature to see if it was having the intended impact or causing any confusion (he fixed a few visual regressions as a result of this).
+- used the [user interview app](/docs/apps/user-interview) to prompt users who tried sampling for a user interview.
+
+![Dashboard](../images/blog/measuring-feature-success/dashboard.png)
+
+After doing all these, he was confident the feature was trending towards success and continued the rolling out until he hit 100% of users.
+
+### 4. Continued development post-launch
+
+Success after launch looks like continued growth in usage metrics. It also looks like recommendations or public praise for that feature. If the feature has a large enough impact, it may even inspire a marketing push or case study. 
```

Comment:
```suggestion
Success after launch looks like continued growth in usage metrics. It also looks like recommendations, or public praise for that feature. If the feature has a large enough impact, it may even inspire a marketing push or case study. 
```